src/modules/formBuilder/components/FormBuilder.tsx

```diff
@@ -124,15 +126,24 @@ const FormBuilder: React.FC<FormBuilderProps> = ({
         component='form'
         onSubmit={rhfMethods.handleSubmit(updateForm)}
       >
-        <Box sx={{ flexGrow: 1 }}>
+        <Box
+          sx={{
+            flexGrow: 1,
+            // Can't exceed 100% width of the container minus width of the Palette
+            maxWidth: 'calc(100% - 52px)',
```

Comment:
This is to prevent the long labels causing horizontal scroll. Normal pages that use `DashboardContentContainer` don't need this because of the max width set [here](https://github.com/greenriver/hmis-frontend/blob/ced6ce01206afe925d8005af6251af9d3f89bfaf/src/components/layout/dashboard/DashboardContentContainer.tsx#L75), I think. But we circumvented that in the FormBuilder in order to pin the Palette on the right, as described below.

Let's discuss: it's a bit of a code smell to have `52px` hard coded here -- it comes from the current width of the palette items + their padding, which could change.

Reply:
Yeah good point. If we need to have a fixed width like this, I think its better if we pull it out into a constant `PALETTE_WIDTH`, and use that constant to fix the actual width on the palette, too. That way, we (hopefully) won't run into the scenario where the palette width inadvertently changes, and this 52px gets out of sync.